modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/taglib/servlet/taglib/AccountEntryCommerceChannelDefaultsScreenNavigationEntry.java

```diff
@@ -0,0 +1,161 @@
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.account.web.internal.frontend.taglib.servlet.taglib;
+
+import com.liferay.account.constants.AccountConstants;
+import com.liferay.account.model.AccountEntry;
+import com.liferay.account.service.AccountEntryService;
+import com.liferay.commerce.account.constants.CommerceAccountWebKeys;
+import com.liferay.commerce.account.web.internal.constants.AccountEntryScreenNavigationEntryConstants;
+import com.liferay.commerce.account.web.internal.display.context.CommerceAccountDisplayContext;
+import com.liferay.commerce.product.service.CommerceChannelAccountEntryRelService;
+import com.liferay.commerce.product.service.CommerceChannelService;
+import com.liferay.commerce.service.CommerceShippingMethodService;
+import com.liferay.commerce.service.CommerceShippingOptionAccountEntryRelService;
+import com.liferay.commerce.shipping.engine.fixed.service.CommerceShippingFixedOptionService;
+import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationEntry;
+import com.liferay.frontend.taglib.servlet.taglib.util.JSPRenderer;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermission;
+import com.liferay.portal.kernel.service.UserService;
+import com.liferay.portal.kernel.util.Portal;
+
+import java.io.IOException;
+
+import java.util.Objects;
+
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+/**
+ * @author Pedro Victor Silvestre
+ */
+@Component(
+	property = "screen.navigation.entry.order:Integer=10",
+	service = ScreenNavigationEntry.class
+)
+public class AccountEntryCommerceChannelDefaultsScreenNavigationEntry
+	extends AccountEntryCommerceChannelDefaultsScreenNavigationCategory
+	implements ScreenNavigationEntry<AccountEntry> {
+
+	@Override
+	public String getEntryKey() {
+		return AccountEntryScreenNavigationEntryConstants.
```

Comment:
@daiazius Please make `getEntryKey()` delegate to `getCategoryKey()` when they return the same constant.

Reply:
@dantewang I asked them to use constants, as using delegate will introduce another reference, and it does not make sense to have that reference.